src/main/java/uk/gov/justice/hmpps/prison/web/config/UrlRewriteConfig.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.justice.hmpps.prison.web.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Component;
+import org.tuckey.web.filters.urlrewrite.Conf;
+import org.tuckey.web.filters.urlrewrite.UrlRewriteFilter;
+
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import java.io.IOException;
+
+@Component
+public class UrlRewriteConfig extends UrlRewriteFilter {
```

Comment:
Just wondering if there is any value in overriding anything else here to see when this is used/called for tracing purposes?

e.g. the doFilter(...) method.

or using aspects or something to add some logging perhaps?